Modules/posixmodule.c

```diff
@@ -13996,6 +13996,82 @@ os_waitstatus_to_exitcode_impl(PyObject *module, PyObject *status_obj)
 }
 #endif
 
+#ifdef MS_WINDOWS
+static PyObject *
+windows_namespace(void)
+{
+    PyObject *d = PyDict_New(), *ret = NULL, *consts = NULL, *errors = NULL;
+    if (d == NULL) {
+        return NULL;
+    }
+
+#define SET(name) \
+    { \
+    PyObject *o = PyLong_FromLong(name); \
+    if (o == NULL) { \
+        goto error; \
+    } \
+    int res = PyDict_SetItemString(d, #name, o); \
+    Py_DECREF(o); \
+    if (res < 0) { \
+        goto error; \
+    } \
+    } \
+
+    SET(DELETE);
+    SET(READ_CONTROL);
+    SET(WRITE_DAC);
+    SET(WRITE_OWNER);
+    SET(SYNCHRONIZE);
+    SET(STANDARD_RIGHTS_REQUIRED);
+    SET(STANDARD_RIGHTS_READ);
+    SET(STANDARD_RIGHTS_WRITE);
+    SET(STANDARD_RIGHTS_EXECUTE);
+    SET(STANDARD_RIGHTS_ALL);
+    SET(SPECIFIC_RIGHTS_ALL);
+    consts = _PyNamespace_New(d);
```

Comment:
This should also include generic rights and the access-system right:

```C
SET(GENERIC_READ);
SET(GENERIC_WRITE);
SET(GENERIC_EXECUTE);
SET(GENERIC_ALL);
SET(MAXIMUM_ALLOWED);
SET(ACCESS_SYSTEM_SECURITY);
```

The generic file mappings are also common for testing access:

```C
SET(FILE_GENERIC_READ);
SET(FILE_GENERIC_WRITE);
SET(FILE_GENERIC_EXECUTE);
SET(FILE_ALL_ACCESS);
```